### PR TITLE
Completed contacts list UI

### LINKED ISSUE #4: Create list contacts UI

### PR DESCRIPTION
**Pull request by vipul-goel**
Fixes #4

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                    setSelectedContactIds(noteIds);
+                  }
+                }}
+              />
+            </th>
+            <th className="text-left text-gray-400">name</th>
+            <th className="text-left text-gray-400">email</th>
+            <th className="text-center text-gray-400">department</th>
+            <th className="text-center text-gray-400">contact number</th>
+            <th className="text-center text-gray-400">add to basecamp</th>
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx:34]: Please remove `text-gray-400`.

**Review by karthiknmenon — CHANGES_REQUESTED** [review]
@vipul-goel 
Please check the suggestion.

File: app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx (hunk 1 above)
Comment (vipul-goel) [reply app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx:34]: @karthiknmenon  The `text-gray-400` matches with the heading text color in the design. Do you want me to remove it anyway?

Comment (karthiknmenon) [reply app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx:34]: Yes.

**Pull request merged**